<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialogFocus</class>
 <widget class="QWidget" name="SettingsDialogFocus">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>365</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="margin">
    <number>9</number>
   </property>
   <item>
    <widget class="QLabel" name="heading">
     <property name="text">
      <string>Configure Focus Settings</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="preferencesGroup">
     <property name="title">
      <string>Preferences</string>
     </property>
     <layout class="QGridLayout">
      <item row="0" column="0">
       <layout class="QGridLayout">
        <property name="margin">
         <number>0</number>
        </property>
        <property name="spacing">
         <number>9</number>
        </property>
        <item row="1" column="0">
         <widget class="QLabel" name="languageLabel">
          <property name="text">
           <string>Focus Window Height:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="colourLabel">
          <property name="text">
           <string>Focus Window Width:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QSlider" name="focusWindowW">
          <property name="minimum">
           <number>50</number>
          </property>
          <property name="maximum">
           <number>1200</number>
          </property>
          <property name="pageStep">
           <number>10</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QSlider" name="focusWindowH">
          <property name="minimum">
           <number>50</number>
          </property>
          <property name="maximum">
           <number>800</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Focus Window Centre X:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Focus Window Centre Y:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QSlider" name="focusWindowX">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimum">
           <number>50</number>
          </property>
          <property name="maximum">
           <number>1200</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QSlider" name="focusWindowY">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimum">
           <number>50</number>
          </property>
          <property name="maximum">
           <number>900</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Dynamic Pose Window</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QSlider" name="dynamicPoseWindow">
          <property name="minimum">
           <number>100</number>
          </property>
          <property name="maximum">
           <number>1000</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="4" column="2">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>Focus Reset Upper Bound (steps)</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>Focus Reset Lower Bound (steps)</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QSlider" name="focusUpperBound">
          <property name="maximum">
           <number>3200</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QSlider" name="focusLowerBound">
          <property name="minimum">
           <number>-5500</number>
          </property>
          <property name="maximum">
           <number>0</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="5" column="2">
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="6" column="2">
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>378</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>focusWindowW</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label_3</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>223</x>
     <y>72</y>
    </hint>
    <hint type="destinationlabel">
     <x>269</x>
     <y>73</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>focusWindowH</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label_6</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>190</x>
     <y>108</y>
    </hint>
    <hint type="destinationlabel">
     <x>277</x>
     <y>102</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>focusWindowX</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label_4</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>172</x>
     <y>128</y>
    </hint>
    <hint type="destinationlabel">
     <x>277</x>
     <y>133</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>focusWindowY</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label_5</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>187</x>
     <y>159</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>164</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dynamicPoseWindow</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label_8</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>196</x>
     <y>190</y>
    </hint>
    <hint type="destinationlabel">
     <x>266</x>
     <y>190</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>focusUpperBound</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label_11</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>204</x>
     <y>221</y>
    </hint>
    <hint type="destinationlabel">
     <x>270</x>
     <y>221</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>focusLowerBound</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label_12</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>182</x>
     <y>252</y>
    </hint>
    <hint type="destinationlabel">
     <x>271</x>
     <y>252</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
